packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -88,7 +89,10 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
                 value: gloss,
               }))}
               // The extra 24 pixel give room for the padding around the text.
-              style={{ width: width + 24, fontFamily: font }}
+              style={{
+                width: width + 24,
+                fontFamily: expandFontFamily(font ?? 'Noto Serif'),
```

Comment:
Let's fall back to Noto Sans instead.

Reply:
So the default font should also be Noto Sans?

Reply:
That's my thought. It's better for screens. We are using serif for Biblical texts, and sans serif for glosses and UI text